docs/insomnia/declarative-config.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: article-detail
+title:  Declarative Config
+category: "Work with Other Kong Products"
+category-url: kong-products
+---
+
+{:.alert .alert-primary}
+**Note**: This feature is currently only available on Design Documents.
+
+Insomnia creates a preformatted [decK](https://docs.konghq.com/deck) file when you add endpoints to a Document.
+
+Once you have added your endpoints, click on the arrow next to your Document name. In the dropdown menu, click **Declarative Config**.
```

Comment:
```suggestion
Once you have added your OpenAPI spec, click on the arrow next to your Document name. In the dropdown menu, click **Declarative Config**.
```

Reply:
Also possibly worth cross-linking to the `inso generate config` command docs, since that command achieves the same thing

Reply:
Code-wise this is the wrong place for the check to exist, but here's the code that says "only openapi 3.x` is supported for [delcarative](https://github.com/Kong/insomnia/blob/c6a7c4d68272b3c4f3d78d4dc9031ae8e49b3f7a/plugins/insomnia-plugin-kong-declarative-config/src/generate.js#L7-L15) and [k8s](https://github.com/Kong/insomnia/blob/c6a7c4d68272b3c4f3d78d4dc9031ae8e49b3f7a/plugins/insomnia-plugin-kong-kubernetes-config/src/generate.js#L7-L15)